src/Collection.php

```diff
@@ -360,7 +361,8 @@ public function equals(iterable $other): bool
 
     public function every(callable ...$callbacks): bool
     {
-        return (new Every())()(static fn (): bool => false)(...$callbacks)($this)->current();
+        return (new Every())()(static fn (int $index, $value, $key, iterable $iterable) => CallbacksArrayReducer::or()($callbacks)($value, $key, $iterable))(static fn (bool $r): bool => $r)($this)
+            ->current();
```

Comment:
should we not use the `CallbacksArrayReducer` inside the `Every` operation? That would still allow using it with multiple callbacks as a standalone operation

Reply:
Yes ! This is on my todo list indeed.

Reply:
Done.